staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -167,3 +167,59 @@ type TracingConfiguration struct {
 	// Defaults to 0.
 	SamplingRatePerMillion *int32 `json:"samplingRatePerMillion,omitempty" protobuf:"varint,2,opt,name=samplingRatePerMillion"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// FlowControlConfiguration holds flow control parameters.
+type FlowControlConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// WorkEstimatorConfiguration holds work estimator parameters.
+	// +optional
+	WorkEstimatorConfiguration *WorkEstimatorConfiguration `json:"workEstimatorConfig,omitempty"`
+}
+
+// WorkEstimatorConfiguration holds work estimator parameters.
+type WorkEstimatorConfiguration struct {
+	// ListWorkEstimatorConfiguration holds work estimator parameters related to list requests.
+	// +optional
+	ListWorkEstimatorConfiguration *ListWorkEstimatorConfiguration `json:"listWorkEstimatorConfig,omitempty"`
```

Comment:
Including the words "work estimator configuration" in this field name seems more verbose than necessary, since it is in the context of a "work estimator configuration".  Compare, for example, with https://kubernetes.io/docs/reference/config-api/apiserver-config.v1/ --- where the field name is simply `plugins` for the slice of admission plugin configurations.

Reply:
Changed:
- `workEstimatorConfig` to `workEstimator`
- `listWorkEstimatorConfig` to `listRequests`
- `mutatingWorkEstimatorConfig` to `mutatingRequests`

I wonder if we should collapse this structure by removing `ListWorkEstimatorConfiguration` and `MutatingWorkEstimatorConfiguration`.